packages/patterns/src/components/reset-overrides-control.js

```diff
@@ -3,7 +3,7 @@
  */
 import {
 	store as blockEditorStore,
-	BlockControls,
+	__unstableBlockToolbarLastItem as BlockToolbarLastItem,
```

Comment:
It looks like we're not currently using that for anything. 🤔

Reply:
It was introduced in #39290.

Reply:
Maybe @youknowriad has an opinion here (I think you added the grouping once upon a time). But I'm not against this approach.

Reply:
You're right it is not used anymore 🤷 It seems it was also used in [this pull request](https://github.com/WordPress/gutenberg/pull/46335), and the initial implementation was "removed" in [this other one](https://github.com/WordPress/gutenberg/pull/46335).